stories/checkbox.stories.js

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
What is the reason for adding this file to this PR? It isn't mentioned in the PR description or the ACs/IBs of the issue so I'm not sure why it's here.

Reply:
was being a bit of a boy scout. noticed it wasn't there I did mention in the channel whether I should add or not since it was missed out. I can remove and open a different PR if you like?

Reply:
I think it should be removed to keep this PR focused on the `className` change at hand for now. We don't really use the storybook much aside from visual regression testing (VRT), so I don't know if there's much value in extending it right now, but if you feel it's worth adding then I'd say it's a separate issue 😄